yb-voyager/cmd/importDataFileCommand.go

```diff
@@ -309,8 +309,8 @@ func init() {
 	}
 
 	importDataFileCmd.Flags().StringVar(&fileTableMapping, "file-table-map", "",
-		"mapping between file name in 'data-dir' to table name.\n"+
-			"Note: default will assume the file names in format as mentioned in the docs. Refer - link") // TODO: if not given default should be file names
+		"comma separated list of mapping between file name in 'data-dir' to a table in database.\n"+
+			"Note: default will be to import all the files in --data-dir with given format, for eg: table1.csv to import into table1 in database.")
```

Comment:
default will be to import all the files in 'data-dir' with the given format. For eg. table1.....
(As an alternative for the 'data-dir' change, you can instead choose to change the earlier reference from `'data-dir'` to `--data-dir`)